services/src/unit-test/java/uk/gov/moj/sdt/services/UpdateRequestServiceTest.java

```diff
@@ -63,45 +63,67 @@
 import uk.gov.moj.sdt.utils.AbstractSdtUnitTestBase;
 import uk.gov.moj.sdt.utils.SdtContext;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
 /**
  * Test class for Update Request Service.
  *
  * @author Manoj Kulkarni
  */
-public class UpdateRequestServiceTest extends AbstractSdtUnitTestBase {
-    /**
-     * Logger for debugging.
-     */
-    @SuppressWarnings("unused")
-    private static final Logger LOGGER = LoggerFactory.getLogger(UpdateRequestServiceTest.class);
+@ExtendWith(MockitoExtension.class)
+class UpdateRequestServiceTest extends AbstractSdtUnitTestBase {
 
     /**
      * Mocked Individual Request Dao object.
      */
+    @Mock
     private IIndividualRequestDao mockIndividualRequestDao;
 
     /**
-     * Update Request Service object.
+     * Mocked messaging utility.
      */
-    private UpdateRequestService updateRequestService;
+    @Mock
+    private IMessagingUtility mockMessagingUtility;
 
     /**
-     * Mocked messaging utility.
+     * Update Request Service object.
      */
-    private IMessagingUtility mockMessagingUtility;
+    private UpdateRequestService updateRequestService;
+
+    private static final String BULK_SUBMISSION_STATUS_IS_INCORRECT = "Bulk submission status is incorrect";
+
+    private static final String BULK_SUBMISSION_COMPLETED_DATE_SHOULD_BE_POPULATED =
+            "Bulk submission completed date should be populated";
+
+    private static final String BULK_SUBMISSION_UPDATED_DATE_SHOULD_BE_POPULATED =
+            "Bulk submission updated date should be populated";
+    private static final String INDIVIDUAL_REQUEST_STATUS_IS_INCORRECT = "Individual request status is incorrect";
+
+    private static final String MCOL_IREQ_0001 = "MCOL_IREQ_0001";
+
+    private static final String RESPONSE = "response";

```

Comment:
These constant names aren't very descriptive!  "MCOL_IREQ_0001" is used as an SDT request reference and "response" is used as a dummy target response so they should be named after those.

Reply:
renamed to, SDT_REQUEST_REF and DUMMY_TARGET_RESP

Reply:
Confirmed that constants have been renamed.